src/Orchard.Web/Modules/Orchard.Projections/Services/ProjectionManager.cs

```diff
@@ -168,7 +168,8 @@ public class ProjectionManager : IProjectionManagerExtension {
             foreach (var sortCriterion in queryRecord.SortCriteria.OrderBy(s => s.Position)) {
                 var sortCriterionContext = new SortCriterionContext {
                     Query = groupQuery,
-                    State = FormParametersHelper.ToDynamic(sortCriterion.State)
+                    State = FormParametersHelper.ToDynamic(sortCriterion.State),
+                    VersionScope = queryRecord.VersionScope
```

Comment:
Instead of duplicating this property in all context objects, can't we just pass the parent Query which already  has the value?

Reply:
Do you mean to add the QueryPart to the context?